modules/ROOT/pages/kernel-args.adoc

```diff
@@ -0,0 +1,72 @@
+= Modifying Kernel Arguments
+
+Currently to change kernel arguments, you must script a systemd service which runs `rpm-ostree kargs --reboot`. The command supports appending kernel arguments (via `--append KEY[=VAL]`), deleting them (via `--delete KEY[=VAL]`), and replacing them (via `--replace KEY[=VALUE]`).
+
+NOTE: In the future, we will have a more Ignition-friendly method of doing this with stronger guarantees. See https://github.com/coreos/ignition/issues/1051 for more information.
```

Comment:
Would you want to make it a hyperlink? 

```suggestion
NOTE: In the future, we will have a more Ignition-friendly method of doing this with stronger guarantees. See the [upstream issue](https://github.com/coreos/ignition/issues/1051] for more information.
```

Reply:
Hmm, does AciiDoc support this Markdown-like syntax. Does not it have to be [like this](https://docs.fedoraproject.org/en-US/fedora-docs/asciidoc-fedora/markup/#snippets)?

Reply:
ahh, yes I used the wrong syntax:

```suggestion
NOTE: In the future, we will have a more Ignition-friendly method of doing this with stronger guarantees. See the https://github.com/coreos/ignition/issues/1051[upstream issue] for more information.
```

Reply:
Done!